src/Toolkit/Str.php

```diff
@@ -469,7 +469,12 @@ public static function esc(string $string, string $context = 'html'): string
 	public static function excerpt($string, $chars = 140, $strip = true, $rep = ' …')
 	{
 		if ($strip === true) {
-			$string = strip_tags(str_replace('<', ' <', $string));
+			// ensure that opening tags are preceded by a space, so that
+			// when tags are skipped we can be sure that words stay separate
+			$string = preg_replace('#\s*<([^\/])#', ' <${1}', $string);
+
+			// strip tags
```

Comment:
To make the comment a bit more useful than a 1:1 copy of the function name 😅

```suggestion
			// in strip mode, we always return plain text
```

Reply:
Fair enough :D